### ISSUE TITLE
Don't load data until done picking options

### ISSUE THREAD

**Issue by fbertsch** [mozilla/telemetry-dashboard#255]
The evolutions/histograms UI is too aggressive in updating things before I'm done picking the things I want

**Comment by chutten** [comment]
Fair enough. We can probably put a couple-hundred-millisecond delay between input and response.

**Comment by himanish-star** [comment]
@chutten , Can I work on this issue once #375 gets merged ? Or can I start my work simultaneously

**Comment by chutten** [comment]
You can work on this simultaneously, if you'd like! Just be sure to keep it on its own branch and rebase it when #375 is merged.

**Comment by himanish-star** [comment]
@chutten , yeah sure. Thanks

**Comment by kdrosado** [comment]
@chutten , Is this issue still open?  Has it be assigned? If not, could I work on it?

**Comment by chutten** [comment]
It isn't assigned, so it's free to be worked on.

However, it's a complicated one. The interaction model among all of the different multiselects and the use of some unmaintained libraries for them made it rather difficult to work with. 

You're still welcome to take it on, but it might be better to get used to the codebase before diving into this one. What do you think?

**Comment by chutten** [comment]
@Sylvia23 has offered to take this on, and knows enough about the codebase to understand how complex it's likely to be.
